app/controllers/api/stateless/braintree/subscriptions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module Stateless
+    module Braintree
+      class SubscriptionsController < StatelessController
+        before_filter :authenticate_request!
+
+        def index
+          @subscriptions = subscriptions_for_member(@current_member)
```

Comment:
There's a duplicate `customer` method for these. Should we extract the service class into its own file?